plugins/inputs/system/linux_sysctl_fs.go

```diff
@@ -23,6 +24,9 @@ func (_ SysctlFS) Description() string {
 func (_ SysctlFS) SampleConfig() string {
 	return sysctlFSSampleConfig
 }
+func OSGetEnv(key string) string {
```

Comment:
These functions don't really encapsulate anything, you could remove them or maybe you could have them return the default value when unset and call it something like `GetHostProc() string` or `GetHostSys() string`.